pod.go

```diff
@@ -191,8 +191,12 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 	addCounter := func(desc *prometheus.Desc, v float64, lv ...string) {
 		addConstMetric(desc, prometheus.CounterValue, v, lv...)
 	}
+	for _, containerStatus := range p.Status.Conditions {
+		if containerStatus.Type == "Initialized" {
+			addGauge(descPodInfo, 1, p.Status.HostIP, p.Status.PodIP, containerStatus.LastTransitionTime.String(), nodeName, extractCreatedBy(p.Annotations))
```

Comment:
@zouyee 

* Does the `Initialized` status indicates that the pod has started?
* IIUC, there should only be one `Initialized` condition, so we can `break` once we find it.